src/internal/signalr/client.go

```diff
@@ -165,14 +176,25 @@ func (c *client) invoke(method string, args ...any) error {
 	timer := time.NewTimer(c.cfg.GetSignalRInvokeTimeout())
 	defer timer.Stop()
 
-	results := c.c.Invoke(method, args...)
+	bckoff := c.exponentialBackoff()
+
+	for i := 0; i < c.cfg.GetSignalRInvokeRetryCount(); i++ {
+		results := c.c.Invoke(method, args...)
+
+		select {
+		case result := <-results:
+			return result.Error
+		case <-timer.C:
```

Comment:
so you have a 30 second timeout timer and do inside backoff for 15 minutes?

Reply:
Do I read this code correctly? If you set up a connection and then try to subscribe, but fail, and fail for all backoffs and what not, you will no longer retry...

So if subscribing was impossible because let's say charger was offline for 2 hours, after 2 hours you won't resubscribe, right?